tests/integration/security/webhook/webhook_test.go

```diff
@@ -56,6 +56,11 @@ func TestWebhookManagement(t *testing.T) {
 	framework.
 		NewTest(t).
 		Run(func(ctx framework.TestContext) {
+			cfg := inst.Settings()
+			if cfg.IsIstiodEnabled() {
```

Comment:
@lei-tang fyi.

Reply:
When istiod is enabled, users are unable to manage webhooks through operator?

Reply:
This is only temporary. We need to update operator/istioctl to skip the k8s secret with istiod.